Demo/Demo/SwiftUIComponents/PayPalWebPayments/PayPalWebButtonsView.swift

```diff
@@ -27,15 +27,15 @@ struct PayPalWebButtonsView: View {
 
                 switch selectedFundingSource {
                 case .paypalCredit:
-                    PayPalCreditButton.Representable(color: .gold, size: .full) {
+                    PayPalCreditButton.Representable(color: .gold, size: .regular) {
                         payPalWebViewModel.paymentButtonTapped(funding: .paypalCredit)
                     }
                 case .paylater:
-                    PayPalPayLaterButton.Representable(color: .silver, edges: .softEdges, size: .full) {
+                    PayPalPayLaterButton.Representable(color: .silver, edges: .softEdges, size: .regular) {
```

Comment:
Since the [PR](https://github.com/paypal/paypal-ios/pull/243) that removed several colors has already been merged, can we update this color?

Reply:
Updated